cPP_DSC.adoc

```diff
@@ -1308,46 +1244,56 @@ FCS_RBG.1.1:: The TSF shall perform deterministic random bit generation services
+
+ISO/IEC 18033-3:2010 (Sub Clause 4.5) [HIGHT]
+
+ISO/IEC 29192-2:2019 (Sub Clause 6.3) [LEA]
 
 |===
 
 
-FCS_RBG.1.2:: The TSF shall use a [*selection*: _TSF noise source_ [*assignment*: _name of noise source_], _TSF interface for seeding_] for initialized seeding.
+FCS_RBG.1.2:: The TSF shall use a [selection: _TSF noise source [assignment: name of noise source], TSF interface for seeding_] for initialized seeding.
 
-FCS_RBG.1.3:: The TSF shall update the RBG state by [*selection*: _reseeding, uninstantiating and re-instantiating_] using a [*selection*: _TSF noise source_ [*assignment*: _name of noise source_], _TSF interface for seeding_] in the following situations: [*selection*: 
+FCS_RBG.1.3:: The TSF shall update the RBG state by [selection: _reseeding, uninstantiating and re-instantiating_] using a [selection: _TSF noise source [assignment: name of noise source], TSF interface for seeding_] in the following situations: [selection:
 +
 * _never_, 
 * _on demand,_  
-* _on the condition: [*assignment*: condition],_ 
-* _after [*assignment*: time as supplied according to FPT_STM_EXT.1]_] 
+* _on the condition: [assignment: condition],_ 
+* _after [assignment: time as supplied according to FPT_STM_EXT.1]_] 
 +
-in accordance with [*assignment*: _list of standards_]. 
+in accordance with [assignment: _list of standards_]. 
 
 _Application Note {counter:remark_count}_:: _No rationale is acceptable for not satisfying one of these dependencies._
 +
-_If reseeding is not feasible, the TSF will uninstantiate RBGs, rather than produce output that is of insufficient quality. The listed standards should specify the reseed interval, and procedure for uninstantiating and reseeding. The remaining selection allows the PP Author to require application-specific conditions for reseeding._
+_If a reseeding is selected in the first selection and something other than “never” is selected in the third selection, but reseeding is not feasible, the TSF will uninstantiate RBGs, rather than produce output that is of insufficient quality. The listed standards should specify the reseed interval and procedure for uninstantiating and reseeding. The remaining selection allows the PP Author to require application-specific conditions for reseeding._
 +
 _"Uninstantiate" means that the internal state of the DRBG is no longer available for use._
 +
-_In the second selection, "on demand" means, that a TOE presents an interface to reseed as a TSFI (e.g., an API call). The interface causes the DRBG to reseed at the request of an authorized user, either with an internal source, an external source, or from input provided through the TSFI (e.g., the API call)._
+_In the third selection, "on demand" means, that a TOE presents an interface to reseed as a TSFI (e.g., an API call). The interface causes the DRBG to reseed at the request of an authorized user, either with an internal source, an external source, or from input provided through the TSFI (e.g., the API call)._
```

Comment:
It's the third selection in FCS_RBG.1.3 where the "on demand" is listed. I have updated the text to be clear about what it is pointing to, but that is why.

Reply:
I think we removed a selection ("never"?) which through the count off.  Catalog has been corrected.

Reply:
@slpotte I assume when you say "corrected" you mean you included the clarification ("In the third selection for FCS_RBG.1.3")? Because the text as written is actually correct.

Reply:
I don't think "never" should be removed. This is good and I am leaving it this way for now.